tests/irepela/homework_10/test_context_manager_timer.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+from irepela.homework_10.context_manager_timer import Timer
+
+
+def test_timer():
+
+    with Timer() as timer:
+        time.sleep(3)
+
+    assert timer.total >= 3
+
+    with timer:
+        time.sleep(2)
+
+    assert timer.total >= 2
```

Comment:
in your case it is >= 5

Reply:
I'm now resetting it on enter - so should be as is assert timer.total >= 2